### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Anna-Kuzmenko**
[DEMO LINK](https://Anna-Kuzmenko.github.io/react_goods-selector/)

### CODE REVIEW

**Review by iHateAntichrist — APPROVED** [review]
good clean code

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +16,84 @@ export const goods = [
...
+        { selectedGood
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              {`${selectedGood} is selected`}
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={this.handleRemove}
```
Comment (SerhiiKirik) [inline src/App.tsx:47]: Let's make a destructuring from this in `render` method

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +16,84 @@ export const goods = [
...
-          </td>
-        </tr>
+        <table className="table">
+          <tbody>
+            {goods.map(good => (
+              <tr
+                data-cy="Good"
+                key={good}
+                className={classNames({
+                  'has-background-success-light': good === selectedGood,
```
Comment (SerhiiKirik) [inline src/App.tsx:60]: You did this twice, so, DRY
Move this to the variable
```suggestion
                  'has-background-success-light': ...,
```

File: src/App.tsx [hunk 3]
```diff
@@ -15,71 +16,84 @@ export const goods = [
...
+                      >
+                        -
+                      </button>
+                    ) : (
+                      <button
+                        data-cy="AddButton"
+                        type="button"
+                        className="button"
+                        onClick={() => {
+                          this.setState({ selectedGood: good });
```
Comment (SerhiiKirik) [inline src/App.tsx:80]: Move this to the separate fucntion

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Well done 🚀 
Let's improve your solution 🧑‍💻

**Comment by Anna-Kuzmenko** [comment]
Fixed all )

File: src/App.tsx [hunk 4]
```diff
@@ -15,71 +16,91 @@ export const goods = [
...
-            </button>
-          </td>
+    return (
+      <main className="section container">
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+        { selectedGood
```
Comment (SerhiiKirik) [inline src/App.tsx:43]:
```suggestion
        {selectedGood
```

File: src/App.tsx [hunk 5]
```diff
@@ -15,71 +16,91 @@ export const goods = [
...
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+        <table className="table">
+          <tbody>
+            {goods.map(good => {
+              const isSelected = good === selectedGood;
```
Comment (SerhiiKirik) [inline src/App.tsx:61]:
```suggestion
              const isGoodSelected = good === selectedGood;
```

**Review by SerhiiKirik — APPROVED** [review]
LGTM 🔥

**Pull request closed without merging**